index.js

```diff
@@ -50,13 +50,13 @@ module.exports = {
         type: 'input',
         name: 'subject',
         message: dedent`
-          Write a short, imperative tense description of the change:
+          Write a short, imperative tense description of the change\n  E.g. Fix format in for loop challenge:
```

Comment:
Yes, type is asked for above, this is feat, fix, chore, etc.
subject should be a imperative description of the change. This should not include the `type` or scope already recorded above.

You example includes the type in the subject. So we end up with `fix: Fix format in for loop challenge` which makes the second fix redundant.

A better example for a subject: `changed loop challenge format`. With type fix we get `fix: changed loop challenge format` as the end result